components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadsFeature.kt

```diff
@@ -58,8 +60,11 @@ class DownloadsFeature(
     private val downloadManager: DownloadManager = AndroidDownloadManager(applicationContext),
     private val customTabId: String? = null,
     private val fragmentManager: FragmentManager? = null,
+    private val promptsStyling: PromptsStyling? = null,
     @VisibleForTesting(otherwise = PRIVATE)
-    internal var dialog: DownloadDialogFragment = SimpleDownloadDialogFragment.newInstance()
+internal var dialog: DownloadDialogFragment = SimpleDownloadDialogFragment.newInstance(
```

Comment:
nit: indentation

Reply:
Also, can this be a `val`?

Reply:
It cannot be due to the way `override fun start()` handles it a few lines beneath this.